src/dialogs/vkbasalt.py

```diff
@@ -0,0 +1,340 @@
+                and self.btn_lut_file_path is False:
+
+                self.btn_save.set_sensitive(False)
+        else:
+            self.btn_save.set_sensitive(True)
+
+    def __change_edge_detection_type(self, widget, edge_detection_type):
+        self.smaa_edge_detection = edge_detection_type
+        self.toggle_luma.handler_block_by_func(self.__change_edge_detection_type)
+        self.toggle_color.handler_block_by_func(self.__change_edge_detection_type)
+        if edge_detection_type == "luma":
+            self.toggle_color.set_active(False)
+            self.toggle_luma.set_active(True)
+        elif edge_detection_type == "color":
+            self.toggle_color.set_active(True)
+            self.toggle_luma.set_active(False)
+
+        self.toggle_luma.handler_unblock_by_func(self.__change_edge_detection_type)
+        self.toggle_color.handler_unblock_by_func(self.__change_edge_detection_type)
+
+    def __import_clut(self, *args):
+        def set_path(_dialog, response, _file_dialog):
+            if response == -3:
+                self.btn_lut_file_path = _file_dialog.get_file().get_path()
+
+                if self.btn_lut_file_path.split(".")[-1] == "png":
+
+                    def error_dialog(title, message):
+                        dialog = Adw.MessageDialog.new(self.window, title, message)
+                        dialog.add_response("cancel", "Close")
+                        dialog.present()
+
+                    try:
+                        texture = Gdk.Texture.new_from_filename(self.btn_lut_file_path)
+
+                        width = texture.get_width()
+                        height = texture.get_height()
+
+                        def set_lut_file_path():
+                            if self.action_clut.get_subtitle():
+                                self.btn_lut_file_path = self.action_clut.get_subtitle()
+                            else:
+                                self.btn_lut_file_path = False
+
+                        if " " in self.btn_lut_file_path:
+                            error_dialog(
+                                _("Spaces in File Name"),
+                                _("Color Lookup Table path must not contain any spaces. Please rename the file to remove all spaces.")
+                                )
+                            set_lut_file_path()
+                        elif width != height:
+                            error_dialog(
+                                _("Invalid Image Dimension"),
+                                _("The height and width of the image must be equal.")
+                                )
+                            set_lut_file_path()
+                        else:
+                            self.action_clut.set_subtitle(self.btn_lut_file_path)
+                            self.btn_lut_reset.show()
+
+                        self.__check_state()
+
+                    except GLib.Error:
```

Comment:
I suggest to do `except GLib.Error as err` and then do `logging.error("Could not do XXX: ", err.message)` (or whatever fn you use to debug to the cli). I think that logging.debug does NOT accept f-strings.

Reply:
Is this good? https://github.com/bottlesdevs/Bottles/pull/1668/commits/2a980eb741322dc966982c374e381bbc8cdafb4f

Sadly I can't test it because I can't think of a situation where this error will even happen.

Reply:
I tested this by opening the file chooser and deleting the file before selecting it (the file chooser didn't update with the files removal) and I got this error instead:
```
18:03:38 (ERROR) Color Lookup Table path must not contain any spaces. Please rename the file to remove all spaces. 
Traceback (most recent call last):
  File "/app/share/bottles/bottles/dialogs/vkbasalt.py", line 283, in set_path
    self.btn_lut_file_path = _file_dialog.get_file().get_path()
AttributeError: 'NoneType' object has no attribute 'get_path'
```

Reply:
I wasn't able to reproduce it myself, as the file picker updates automatically. I just committed https://github.com/bottlesdevs/Bottles/pull/1668/commits/965398842b6e49f3cc511368f8fe36f6e0e91383, can you try again?

Reply:
It's the same error except the `Color Lookup Table path must not contain any spaces` message doesn't apper

Reply:
Does the file have any spaces? If so, it's normal. The image name should not contain any spaces, and the size of the image must have the same height and width.

Reply:
No both files I tested do not contain any spaces and they both had the same height and width

Reply:
Seems like it works now. @A6GibKm `except GLib.Error as err` doesn't work. I used `AttributeError` instead.